edgexfoundry/edgex-compose
# build: Change eKuiper to version 1.11

**If your build fails** due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/edgex-compose/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

  <link to docs PR>


## Testing Instructions
run `make run ds-virtual mqtt-broker`
Create eKuiper stream
Create eKuiper rule to post result to MQTT on `result` topic
Use Mqtt.fx to subscribe to `result` topic and verify receiving data
